plugins/mollie/README.md

```diff
@@ -0,0 +1,33 @@
+FoodsoftMollie
+==============
+
+This project adds support for various online payment methods using Mollie to Foodsoft.
+
+* Make sure the gem is uncommented in foodsoft's `Gemfile`
+* Enter your Mollie account details in `config/app_config.yml`
+
+```yaml
+  use_mollie: true
+  # Mollie payment settings
+  mollie:
+    # API key for account: 1234567, website profile: FooInc
+    api_key: test_1234567890abcdef1234567890abcd
+    # Transaction fee as provided by mollie api
+    charge_fees: true
+    currency: EUR # should match the foodcoop's currency
+```
+
+The transaction fee is sub will be added on each payment when set; it is not set by default,
```

Comment:
What does "is sub" mean? (could it be that it used to be "is substracted from ...").
Also, "it is not set by default" -> "it is disabled by default" sounds clearer.